addons/common/XEH_postInit.sqf

```diff
@@ -244,6 +244,35 @@ if (_currentVersion != _previousVersion) then {
 
 call FUNC(checkFiles);
 
+// Let the server know that we have ACE loaded
+missionNamespace setVariable [QGVAR(aceLoaded_) + getPlayerUID player, true, 2];
+
+// This handles clients joining that do not have ACE loaded
+if (isServer) then {
+    addMissionEventHandler ["PlayerConnected", {
+        [{
+            params ["", "_uid", "", "", "_owner"];
+
+            // If ACE is not loaded, kick unit
+            if !(missionNamespace getVariable [QGVAR(aceLoaded_) + _uid, false]) then {
+                // Send function to client
+                _owner publicVariableClient QFUNC(errorMessage);
+
+                // Wait for function to broadcast, then kick client
+                [{
+                    ["[ACE] ERROR", "ACE is not present", {findDisplay 46 closeDisplay 0}, {findDisplay 46 closeDisplay 0}, nil, [true, false]] remoteExecCall [QFUNC(errorMessage), _this];
```

Comment:
This won't work if ACE isn't loaded on the client, only if it's a version from before this PR.